docs/userguide/mng-storage-driver.md

```diff
@@ -0,0 +1,145 @@
+<!--[metadata]>
++++
+title = "Choose a storage driver"
+description = "Learn how select the proper storage driver for your container."
+keywords = ["container, storage, driver, AUFS, btfs, devicemapper,zvfs"]
+[menu.main]
+parent = "mn_use_docker"
++++
+<![end-metadata]-->
+
+# Choose a storage driver
+
+When you start the Docker `daemon`, you can specify which storage driver you want to use with your containers. A container's storage driver can affect the performance a containerized applications. So it is important to understand the different storage options available and select the right one for your application.
```

Comment:
"A container's storage driver" - would that confuse people? because storage drivers are a global setting in docker, and it's not a property/setting of an individual container (nor is the container aware of it)